### PR TITLE
Color tweeks

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
So... yeah. Turns out that changing the name was a lot more complicated once I cleared the cache. Anyhow, fixed a number of issues discovered related to changing the name of the bundle. 

Also, tweeked the UIColor theme stuff to make it easier to remain reverse-compatible to CRF and assorted other fixed discovered during testing of CRF.

### CODE REVIEW

File: ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift [hunk 1]
```diff
@@ -35,65 +35,146 @@ import UIKit
...
+    /// Background color for views that should have a light background.
+    ///
+    /// Default = `UIColor.white`.
+    @objc open class var appBackgroundLight: UIColor {
+        return UIColor.white
+    }
+    
+    /// Background color for views that should have a dark background.
+    ///
+    /// Default = `UIColor.primaryTintColor`
```
Comment (Erin-Mounts) [inline ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift]: .

File: ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift [hunk 2]
```diff
@@ -35,65 +35,146 @@ import UIKit
...
     /// separators.
+    ///
+    /// Default = #EDEDED (237, 237, 237).
     @objc open class var appVeryLightGray: UIColor {
         return UIColor(white: 237.0 / 255.0, alpha: 1.0)
     }
     
     
-    // MARK: App background - default colors
+    /// MARK: Status colors
```
Comment (Erin-Mounts) [inline ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift]: three /'s on a MARK: comment?

File: ResearchStack2UI/ResearchStack2UI/iOS/RSDStepViewController.swift [hunk 3]
```diff
@@ -228,6 +228,21 @@ open class RSDStepViewController : UIViewController, RSDStepViewControllerProtoc
     /// A footer view that includes navigation elements.
     @IBOutlet open var navigationFooter: RSDNavigationFooterView?
     
+    /// The label for displaying step detail text.
```
Comment (Erin-Mounts) [inline ResearchStack2UI/ResearchStack2UI/iOS/RSDStepViewController.swift]: detail --> title (?)

File: ResearchStack2UI/ResearchStack2UI/iOS/RSDStepViewController.swift [hunk 4]
```diff
@@ -228,6 +228,21 @@ open class RSDStepViewController : UIViewController, RSDStepViewControllerProtoc
...
+    open var stepTitleLabel: UILabel? {
+        return self.navigationHeader?.titleLabel ?? self.navigationFooter?.titleLabel
+    }
+    
+    /// The label for displaying step detail text.
```
Comment (Erin-Mounts) [inline ResearchStack2UI/ResearchStack2UI/iOS/RSDStepViewController.swift]: not sure what this should say, but prolly not the same as the doc comment for stepDetailLabel...

File: ResearchStack2UI/ResearchStack2UI/iOS/Views/RSDRoundedButton.swift [hunk 5]
```diff
@@ -78,25 +69,60 @@ import UIKit
...
+    open var usesLightStyle: Bool = false {
         didSet {
-            guard let font = titleFont else { return }
-            titleLabel?.font = font
+            updateColorStyle()
         }
     }
     
-    open var titleColor: UIColor? {
+    /// Should the button display using the "secondard" button style? This style is used for buttons that
```
Comment (Erin-Mounts) [inline ResearchStack2UI/ResearchStack2UI/iOS/Views/RSDRoundedButton.swift]: secondard --> secondary

File: ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift (hunk 2 above)
Comment (syoung-smallwisdom) [reply ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift]: Yup, that includes them in the docs along with the explanatory comment under one of the marks.

**Pull request merged**